amset/scattering/basic.py

```diff
@@ -108,8 +108,8 @@ def from_amset_data(
                 v, (len(amset_data.doping), len(amset_data.temperatures), 1, 1)
             )
 
-            rates = velocities * s_to_au / mfp
-            rates[spin] = rates[..., amset_data.ir_to_full_kpoint_mapping]
+            rates2 = velocities * s_to_au / mfp
```

Comment:
Can you rename this variable `spin_rates` please.

Reply:
Ahh yes I really should get better at naming stuff. Anyhow, all done!